<launch>
	<env name="ROSCONSOLE_CONFIG_FILE" value="$(find map_builder)/config/general.config"/>
	<rosparam file="$(find map_builder)/config/basic_recon_v3.yaml" command="load" ns="" subst_value="true"/>
	<node name="recon_service_node" pkg="map_builder" type="map_builder_service" respawn="false" output="screen"/>
</launch>